crime-commons-schemas/build.gradle

```diff
@@ -0,0 +1,95 @@
+plugins {
+    id "com.javiersc.semver.project" version "0.5.0"
+    id "signing"
+    id "maven-publish"
+    id 'org.jsonschema2dataclass' version '6.0.0'
+}
+
+semver {
+    tagPrefix.set("crime-commons-schemas-")
+}
+
+java {
+    sourceCompatibility = '17'
+}
+
+configurations {
+    compileOnly {
+        extendsFrom annotationProcessor
+    }
+}
+
+def versions = [
+        jacksonVersion   : "2.17.0",
+        validationapiVersion: '2.0.1.Final'
+]
+
+dependencies {
+    implementation "com.fasterxml.jackson.core:jackson-databind:$versions.jacksonVersion"
+    implementation "javax.validation:validation-api:$versions.validationapiVersion"
+}
+
+publishing {
+    publications {
+        maven(MavenPublication) {
+
+            from components.java
+
+            pom {
+                artifactId = project.name
+                name = "Crime Apps Common Schemas"
+                url = "https://github.com/ministryofjustice/laa-crime-commons"
+                description = "A library of shared Crime Apps schemas"
+
+                licenses {
+                    license {
+                        name = "MIT"
+                        url = "https://opensource.org/licenses/MIT"
+                    }
+                }
+                organization {
```

Comment:
I would consider modifying the Organisation block to include our team distribution list, maybe something like below?
```
                organization {
                    id=- "laa-crime-apps"
                    name = "Justice Digital - LAA Crime Apps"
                    email = "laa-crime-apps@digital.justice.gov.uk"
                }
```

Reply:
Done

Reply:
Actually not done - id and email aren't valid fields for the `organisation`!